[WIP] Enhace the Docs (#2)
sdumetz/coordinates: README.md

* Call [fromPolar()](/Functions/fromPolar().adoc) - to convert polar coordinates to cartesian coordinates.

### Better Precision

`Coordinates` provides a `class` to seamlessly convert cartesian coordinates to polar coordinates or vice versa.
It uses floating point math to do so with quadrant checking to always give better precision.

### Functions

Refer [Function list](/Functions) to know about the funtions offered by `Coordinates`.

* [fromCartesian()](/Functions/fromCartesian().adoc)

* [fromPolar()](Functions/fromPolar().adoc)

* [getAngle()](Functions/getAngle().adoc)

* [getR()](/Functions/getR().adoc)

* [getX()](/Functions/getX().adoc)

* [getY()](/Functions/getY().adoc)

## Installation

How to install **Coordinates** library? <br>
You can install **Coordinates** in either of the two ways depending on how you are using the Arduino IDE.

* Option 1: If you are working offline using Arduino IDE please follow the
  steps as given in :
  <br>[Install library in Arduino IDE](https://www.arduino.cc/en/guide/libraries#toc2)

* Option 2: If you are working online using [Arduino Web Editor](https://create.arduino.cc/editor) follow these steps:
  * Sign in to your account.

>   * Open or create new **Sketchbook**  in which you wish to use this library.

Reviewer: ```suggestion
  * Open or create a new **Sketch**  in which you wish to use this library.
```
An Arduino program is a "sketch". The folder where all of a user's sketches are normally stored it the "sketchbook".

  * Click on **Library Manager** in **Libraries** tab.

  * Choose your board and the topic (*optional*)

  * Search for '*Coordinates*', *star* it and now click **Done**.

  * Go to **Favorites** section inside the **Libraries** tab and search for '*Coordinates*'.

>   * Hover the mouse over it and click **Include** or just type "`#include <Coorinates.h>`"

Reviewer 2: typo on this line
```suggestion
  * Hover the mouse over it and click **Include** or just type "`#include <Coordinates.h>`"
```

Author: @sdumetz Thank you for pointing out! Fixed it.

Author: @sdumetz  I need your help for squashing commits!

Reviewer 2: If you're using command line, `git rebase -i <remote>` Remote being the branch you want to squash against. Here it's my master branch so if it's not already in your repo's remotes you must add it first : 
```
git remote add reference git@github.com:sdumetz/coordinates.git
git rebase -i reference/master
```
It then propose you to `pick` or squash each commits individually. In your case you'd want to squash all commits into one so keep the first one as `pick` and every other as  `squash`.

If you'd like to know more about commit squashing, github has a tutorial about it : https://github.blog/2016-04-01-squash-your-commits/

Apart from command line, I don't really know the other options to do it, unfortunately. But I can always do it myself if you want and it really is not much trouble.

Author: Hey @sdumetz thanks for your help! I learnt something new! But I had squashed like this
`git reset --soft HEAD~3 &&
git commit`
referring to [this stackoverflow answer](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git). Is everything fine or is there anything else I have to do?
Sorry for the trouble!

Reviewer 2: No it looks about right. It's often the case with git that there is a lot of ways to make the exact same thing.

Maybe use a more descriptive commit message if you want it to be "perfect", but I'm nitpicking :smile: 

Let me know when you're done with your work and want me to merge this.

Author: Cool! 😄
There is nothing else in my mind apart from this. 
@per1234  What do you think ? What else can we add before merging to make it even better?

>      in your 'Sketchbook'.

Reviewer: ```suggestion
     in your 'Sketch'.
```
